docs/reference.rst

```diff
@@ -298,6 +298,17 @@ For more information about using the streaming support of RAUC, refer to
   This option can be used to set the path of the CA certificate which should be
   used instead of the system wide store of trusted TLS/HTTPS certificates.
 
+``send-headers``
+  This option takes a list of information to send as HTTP header fields.
```

Comment:
This should mention `;` as the list element separator.

Reply:
Yes, will add